web/util/entityPaths.ts

```diff
@@ -1,23 +1,19 @@
 import urlJoin from 'proper-url-join';
-import { Session } from '../types/Session';
-import { Venue } from '../types/Venue';
-import { Person } from '../types/Person';
-
-type Entity = Person | Session | Venue;
 
 export const mainEventId = 'aad77280-6394-4090-afad-1c0f2a0416c6';
 
-export const getEntityPath = (entity?: Entity) => {
+export const getEntityPath = (entity?: any) => {
```

Comment:
If the query is adjustes per the comment above, then instead of `any`, could this be defined as a simpler object type along the lines of `{_type: string; slug?: Slug;}` or some such?

Reply:
3677fcb makes sense with the comment above. :100: